app/addons/dashboard/components.react.jsx

```diff
@@ -0,0 +1,190 @@
+      this.state.clearPolling();
+      activeTaskList.off('change', this.onChange, this);
+    },
+
+    onChange: function () {
+      this.setState(this.getStoreState());
+    },
+
+    render: function () {
+      var collection = this.state.collection;
+      var isEmpty = _.isEmpty(collection);
+      return (
+        <div>
+          <ActiveTaskWidget collection={collection} isEmpty={isEmpty}/>
+        </div>
+        );
+    }
+  });
+
+  var ActiveTaskWidget = React.createClass({
+
+    render: function () {
+      var collection = this.props.collection;
+      var isEmpty = this.props.isEmpty;
+      return (
+        <div className="widget-container active-task-box">
+          <div className="widget-header">
+            <span>Active Tasks</span>
+          </div>
+          <hr className="widget-header-separator"/>
+          <div className="active-task-table">
+            <ActiveTaskTable collection={collection} isEmpty={isEmpty}/>
+          </div>
+        </div>
+        );
+    }
+  });
+
+  var ActiveTaskTable = React.createClass({
+
+    getStoreState: function () {
+      return {
+        filteredTable: activeTaskList.getFilteredTable(this.props.collection)
+      };
+    },
+
+    getInitialState: function () {
+      return this.getStoreState();
+    },
+
+    componentWillReceiveProps: function (nextProps) {
+      this.setState({
+        filteredTable: activeTaskList.getFilteredTable(this.props.collection)
+      });
+    },
+
+    createRows: function () {
+      var isThereResults = this.props.isEmpty;
+      if (this.state.filteredTable.length === 0) {
+        return isThereResults ? this.noActiveTasks() : this.noActiveTasksMatchFilter();
+      }
+
+      return _.map(this.state.filteredTable, function (item, iteration) {
+        return <ActiveTaskTableBodyContents key={Math.random()} item={item} />;
+      });
+    },
+
+    noActiveTasks: function () {
+      return (
+        <tr>
+          <td className="noResult">No active tasks.</td>
+        </tr>
+        );
+    },
+
+    noActiveTasksMatchFilter: function () {
+      return (
+        <tr>
+          <td className="noResult">No active tasks match with filter.</td>
+        </tr>
+        );
+    },
+
+    render: function () {
+      var rows = this.createRows();
+      return (
+        <table className="widget-activeTask-table">
+          <thead>
+            <tr>
+              <th className="pid-column">PID</th>
+              <th className="type-column">Type</th>
+              <th className="progress-column">Progress</th>
+              <th className="updateOn-column">Updated On</th>
+            </tr>
+          </thead>
+          <tbody>
+          {rows}
+          </tbody>
+        </table>
+        );
+    }
+  });
+
+  var ActiveTaskTableBodyContents = React.createClass({
+    getInfo: function (item) {
+      return {
+        type: item.type,
+        updated_on: activeTasksHelpers.getTimeInfo(item.updated_on),
```

Comment:
We use camel-case in the JS rather than underscores (even when API JSON content contains underscores or other things for their property names, just FYI).